go/internal/feast/featurestore.go

```diff
@@ -636,26 +649,32 @@ func (fs *FeatureStore) dropUnneededColumns(onlineFeaturesResponse *serving.GetO
 	}
 }
 
-func (fs *FeatureStore) listFeatureViews(allowCache, hideDummyEntity bool) []*FeatureView {
-	featureViews := fs.registry.listFeatureViews(fs.config.Project)
+func (fs *FeatureStore) listFeatureViews(hideDummyEntity bool) ([]*FeatureView, error) {
```

Comment:
How come we have a `FeatureStore.listFeatureViews` but no `FeatureStore.listRequestFeatureViews`?

Reply:
Added.